apps/src/music/views/Callouts.tsx

```diff
@@ -71,12 +71,11 @@ const availableCallouts: AvailableCallouts = {
   'trigger-button-1': {selector: `#${Triggers[0].id}`},
   'toolbox-first-row': {selector: '.blocklyTreeRow'},
   'flyout-first-block': {
-    selector:
-      '.blocklyFlyout:not([style*="display: none;"]) .blocklyDraggable:nth-of-type(1)',
+    selector: '.blocklyFlyout:not([style*="display: none;"]) .blocklyDraggable',
   },
   'toolbox-second-block': {
     selector:
-      '.blocklyFlyout:not([style*="display: none;"]) .blocklyDraggable:nth-of-type(3)',
```

Comment:
One would expect to match on `:nth-of-type(2)`.  Nothing matched on `:nth-of-type(1)` and the first block matched on `:nth-of-type(2)`.